gui/src/app/Stanc/useStanc.ts

```diff
@@ -0,0 +1,67 @@
+import { useCallback, useEffect, useState } from "react";
+import { StancReplyMessage, StancErrors, Replies, Requests } from "./Types";
+import stancWorkerURL from "./stancWorker?worker&url";
```

Comment:
remind me what these magic characters do? `?` means 'instantiate as a web worker' and `&url` means 'get the URL of the deployed asset' or something?

Reply:
A `?worker` says "bundle the file for use in a web worker"
A `?url` says (as you guessed) "rather than import something, import _a url pointing to the deployed asset_"
A `?raw` means "import the inlined raw text of the file" (this is used in the worker itself)

All are documented at https://vitejs.dev/guide/assets

Reply:
If you wouldn't object to just including the reference to that URL above, it'd be a service--the keywords are otherwise non-obvious and googling punctuation is notoriously fraught